### DISCUSSION TITLE
How to Set Custom Band Plan Banner Colors?

### DISCUSSION THREAD

**Discussion by Arrin-KN1E** [AlexandreRouma/SDRPlusPlus#855] [Q&A] [answered]
Hello! I am experimenting with the band plans and I can easily make an allocation in the JSON file, but I don't know what needs to be changed to support custom colors, if even practical. Sorry, I am not very familiar with JSON or serious programming, in general. 
Also, don't worry; any changes are not going into the repository. Just for my experimentation.

**Comment by AlexandreRouma** [comment, marked as answer]
You can change band type colors or declare new band types in the main config file under the "bandColors" section.

**Comment by Arrin-KN1E** [comment]
@AlexandreRouma Thanks for letting me know. I got confused with the default "satellite" type not being in config, and assumed there was more going on. Thanks again!
